### PR TITLE
Added conditional styling for components.

### LINKED ISSUE #89: Change flat buttons to allow different styling.
A. C. 
- allow the flat button component to take props that change its styling.

### PR DESCRIPTION
**Pull request by ringong95**
Closes #89

### CODE REVIEW

File: js/components/PotluckConfirmationInfo/index.js [hunk 1]
```diff
@@ -3,55 +3,68 @@ import { Text, View, Image, ScrollView } from 'react-native';
...
-          <View style={styles.imageContainer}>
-            <Image style={styles.image} source={require('../../../assets/images/southparklastsupper.jpg')} />
-            <View style={styles.textBanner}>
-              <Text style={styles.bannerText}> South Park: The Last Supper</Text>
-            </View>
+  if (buttonType === 'rsvp') {
+    return (
+      <View style={styles.buttonContainer}>
+        <SingleFlatButton width={'half'} style={styles.rsvpButton} title={'Sounds Good'} />
+        <SingleFlatButton width={'half'} style={styles.rsvpButton} title={'Maybe Next Time'} />
```
Comment (BlakeTurner) [inline js/components/PotluckConfirmationInfo/index.js]: Looks like we should also pass titles in as props

File: js/components/PotluckConfirmationInfo/index.js [hunk 2]
```diff
@@ -3,55 +3,68 @@ import { Text, View, Image, ScrollView } from 'react-native';
...
+      </View>
+    );
+  }
+  if (buttonType === 'confirmation') {
+    return <SingleFlatButton title={'Confirm'} color="green" />;
```
Comment (BlakeTurner) [inline js/components/PotluckConfirmationInfo/index.js]: This title should be a prop

File: js/components/PotluckConfirmationInfo/index.js [hunk 3]
```diff
@@ -3,55 +3,68 @@ import { Text, View, Image, ScrollView } from 'react-native';
...
+  }
+  return null;
+};
+
+const PotluckConfirmationInfo = ({ title, color, buttonNeeded = true, buttonType }) => (
+  <View style={styles.container}>
+    <View style={styles.inputContainer}>
+      <ScrollView style={styles.scrollViewContainer}>
+        <View style={styles.imageContainer}>
+          <Image style={styles.image} source={require('../../../assets/images/southparklastsupper.jpg')} />
```
Comment (BlakeTurner) [inline js/components/PotluckConfirmationInfo/index.js]: This image src should be a prop

File: js/components/PotluckConfirmationInfo/index.js [hunk 4]
```diff
@@ -3,55 +3,68 @@ import { Text, View, Image, ScrollView } from 'react-native';
...
+          <View style={styles.textBanner}>
+            <Text style={styles.bannerText}> South Park: The Last Supper</Text>
```
Comment (BlakeTurner) [inline js/components/PotluckConfirmationInfo/index.js]: Prop plz

File: js/components/PotluckConfirmationInfo/index.js [hunk 5]
```diff
@@ -3,55 +3,68 @@ import { Text, View, Image, ScrollView } from 'react-native';
...
+          </View>
+        </View>
+        <View style={styles.infoBox}>
+          <View style={styles.textContainer}>
+            <Text style={styles.colorText}>Date: </Text>
+            <Text>February 14, 2020</Text>
```
Comment (BlakeTurner) [inline js/components/PotluckConfirmationInfo/index.js]: Prop

File: js/components/PotluckConfirmationInfo/index.js [hunk 6]
```diff
@@ -3,55 +3,68 @@ import { Text, View, Image, ScrollView } from 'react-native';
...
-              <Text style={styles.colorText}>Serving Time: </Text>
-              <Text>1:30PM</Text>
-            </View>
-            <View style={styles.textContainer}>
-              <Text style={styles.colorText}>Location: </Text>
-              <Text>Belgrade, Serbia</Text>
-            </View>
+          <View style={styles.textContainer}>
+            <Text style={styles.colorText}>Arriving Time: </Text>
+            <Text> 12:30PM</Text>
```
Comment (BlakeTurner) [inline js/components/PotluckConfirmationInfo/index.js]: Prop

File: js/components/PotluckConfirmationInfo/index.js [hunk 7]
```diff
@@ -3,55 +3,68 @@ import { Text, View, Image, ScrollView } from 'react-native';
...
           </View>
-          <View>
-            <Image style={styles.mapImage} source={require('../../../assets/images/mockMap.png')} />
+          <View style={styles.textContainer}>
+            <Text style={styles.colorText}>Serving Time: </Text>
+            <Text>1:30PM</Text>
```
Comment (BlakeTurner) [inline js/components/PotluckConfirmationInfo/index.js]: Props

File: js/components/PotluckConfirmationInfo/index.js [hunk 8]
```diff
@@ -3,55 +3,68 @@ import { Text, View, Image, ScrollView } from 'react-native';
...
+            <Text>Belgrade, Serbia</Text>
+          </View>
+        </View>
+        <View>
+          <Image style={styles.mapImage} source={require('../../../assets/images/mockMap.png')} />
+        </View>
+        <View style={styles.descriptionBox}>
+          <Text style={styles.descriptionHeader}>Description</Text>
+          <Text style={styles.descriptionText}>
+            Lorem Ipsum is simply dummy text of the printing and typesetting industry.
```
Comment (BlakeTurner) [inline js/components/PotluckConfirmationInfo/index.js]: ...I think you get the picture

**Review by BlakeTurner — CHANGES_REQUESTED** [review]

File: js/components/PotluckConfirmationInfo/index.js (hunk 8 above)
Comment (ringong95) [reply js/components/PotluckConfirmationInfo/index.js]: Prop?

**Review by BlakeTurner — APPROVED** [review]
Nice changes

**Pull request merged**